cmd/yace/main.go

```diff
@@ -170,3 +171,22 @@ func (s *scraper) scrape(ctx context.Context) (err error) {
 	log.Debug("Metrics scraped.")
 	return nil
 }
+
+func (s *scraper) reload(ctx context.Context) func(http.ResponseWriter, *http.Request){
```

Comment:
Lets put this function into the pkg/config.go. Would like to keep main.go small.

Can you please verify if the code was run in gofmt? https://pkg.go.dev/cmd/gofmt

Reply:
Hi thanks for reviewing, yes the code is run in gofmt